Sources/Styleguide/TODO.swift

```diff
@@ -51,12 +51,6 @@ public let select = CssSelector.elem(selectElement)
 private let textareaElement = CssSelector.Element.other("textarea")
 public let textarea = CssSelector.elem(textareaElement)
 
-// this is just a curried version of `li`. i wasnt able to use `curry(li)` cause the compiler was confused by
```

Comment:
How'd this magically start working?

Reply:
I have no idea! I think maybe I just was doing something wrong elsewhere and now I have a better grip on stufff